packages/strapi-helper-plugin/lib/src/components/LabelIconWrapper/index.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const LabelIconWrapper = styled.span`
+  margin-left: 5px;
```

Comment:
Also, margin inside a component often break alignment 😊 . I know that we're going to move from that in the future, but here's an interesting article on "why": https://mxstbr.com/thoughts/margin